ReflectionTest/SomeLibrary/User.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SomeLibrary
+{
+    public class User
+    {
+        internal User() { }
+
+        public User(string login, string password, bool isActive = false)
+        {
+            Login = login;
+            Password = password;
+            IsActive = isActive;
+        }
+
+        public string Login { get; set; }
```

Comment:
Set shouldn't be public

Reply:
Replaced: public string Login { get; private set; }